Packs/Phishing/Scripts/DeleteReportedEmail/README.md

```diff
@@ -31,3 +34,12 @@ Use this script to delete a reported phishing email from the mailbox it was repo
 | DeleteReportedEmail.using_brand | The email service that was used to delete the email. | String |
 | DeleteReportedEmail.email_subject | The subject of the deleted email. | String |
 | DeleteReportedEmail.message_id | The message ID of the deleted email. | String |
+
+## Troubleshooting
+---
+* If the `Reported Email Origin` field is missing or has a value of `None`, the script will not be able to locate the email and fail.  
+  This can happen if the mail to was forwarded to the listener mailbox was not forwarded as an attachment (with an `EML` file) like it should.
```

Comment:
```suggestion
  This can happen if the email forwarded to the listener mailbox was not forwarded as an attachment (with an `EML` file) as it should.
```